binstar_client/utils/config.py

```diff
@@ -25,8 +30,28 @@
 else:
     dirs = AppDirs('binstar', 'ContinuumIO')
 
-SITE_CONFIG = join(dirs.site_data_dir, 'config.yaml')
-USER_CONFIG = join(dirs.user_data_dir, 'config.yaml')
+CONDA_PREFIX = sys.prefix
+try:
+    # We're in the root environment
+    import conda.config
+    CONDA_ROOT = conda.config.root_dir
+except ImportError:
```

Comment:
@kalefranz, is there a reliable way to find the conda root prefix from outside the root environment (preferably without shelling out to conda, as below)?